apis/apis.json

```diff
@@ -111,7 +111,7 @@
       "Microsoft.AspNet.WebApi.Core": "5.2.3",
```

Comment:
Is it worth checking these dependency versions are appropriate? Are there more recent versions that are worth picking up, for example?

Reply:
Good catch, bumped all to the latest stable release, thanks Jon!